### PR TITLE
Filter torrent results to match user preferences

### PR DESCRIPTION
**Pull request by welcoMattic**
Ref #3

### CODE REVIEW

File: infrastructure/docker/docker-compose.yml [hunk 1]
```diff
@@ -60,7 +60,7 @@ services:
             - "traefik.http.routers.${PROJECT_NAME}-redis.tls=true"
 
     jackett:
-        image: linuxserver/jackett
+        image: linuxserver/jackett:development
```
Comment (Korbeil) [inline infrastructure/docker/docker-compose.yml:63]: Really usefull ?

File: project/src/Controller/DashboardController.php [hunk 2]
```diff
@@ -31,11 +31,11 @@ public function index(Request $request, MovieRepository $movieRepository): Respo
                 return $this->redirectToRoute('dashboard');
             }
 
-            $movies = $this->tmdbClient->searchMovie(['query' => $search])->getResults();
-            $tvShows = $this->tmdbClient->searchTvShow(['query' => $search])->getResults();
+            $movies = $this->tmdbClient->searchMovie(['query' => $search, 'language' => 'fr'])->getResults();
```
Comment (Korbeil) [inline project/src/Controller/DashboardController.php:34]: Should be a config var instead of `fr` in every calls

Comment (welcoMattic) [reply project/src/Controller/DashboardController.php:34]: Yes it’s a WIP PR, it’s not yet implemented

File: infrastructure/docker/docker-compose.yml (hunk 1 above)
Comment (welcoMattic) [reply infrastructure/docker/docker-compose.yml:63]: Useful to get latest fix about YGGtorrent

**Pull request closed without merging**